diff --git a/lib/MessageContainer/index.js b/lib/MessageContainer/index.js
index 964b4364a33e47272e009928210be5d01d53ed63..9ac8f2c6c362cc11bb903145958f7d46c5e59a81 100644
--- a/lib/MessageContainer/index.js
+++ b/lib/MessageContainer/index.js
@@ -217,7 +217,7 @@ function MessageContainer(props) {
       {isScrollToBottomEnabled
             ? renderScrollToBottomWrapper()
             : null}
-      <DayAnimated scrolledY={scrolledY} daysPositions={daysPositions} listHeight={listHeight} messages={messages} isLoadingEarlier={isLoadingEarlier}/>
+      <DayAnimated scrolledY={scrolledY} daysPositions={daysPositions} listHeight={listHeight} messages={messages} isLoadingEarlier={isLoadingEarlier} renderDay={props?.renderDay}/>
     </View>);
 }
 export default MessageContainer;
